app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -9,8 +9,16 @@ class HowToVerifyController < ApplicationController
     check_or_render_not_found -> { self.class.enabled? }
 
     def show
+      if idv_session.skip_doc_auth == false
+        @selection = Idv::HowToVerifyForm::REMOTE
+      elsif idv_session.skip_doc_auth == true
+        @selection = Idv::HowToVerifyForm::IPP
+      else
+        @selection = nil
+      end
```

Comment:
@soniaconnolly  I added this logic to show.  If the user goes back (either with back button or with browser back button), the radio button previously selected will still be selected. Good find!  

Here is the commit 20d1410014a9b54aeab980f4311eca378d56040b so you can review all changes that went with this

Reply:
Thanks for fixing! Looks great, with the additional changes we discussed.